keyvault/azure-securtiy-keyvault-jca/run-with-command-line-client-side/README.md

```diff
@@ -1,92 +0,0 @@
-# Use azure-security-keyvault-jca.jar via Command Line in client side
```

Comment:
Why add `keyvault/.gitignore` file? And it's an empty file.

Reply:
`JCA` is not related to spring-boot version. We shouldn't delete related samples.

Reply:
the file keyvault/.gitignore is used to keep the service directory visualization like other directories.

Reply:
JCA sample can not work in Java 17.

Reply:
Why should we keep this directory when the directory is empty?

Reply:
Keep the root page displaying only the service name, which means not displaying the sample project name.
![image](https://user-images.githubusercontent.com/19926623/194806001-f2ae31c6-fede-4784-b4d3-b9cdeb8ccc77.png)